from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(905, 619)
        self.centralwidget = QtWidgets.QWidget(parent=MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.tabWidget = QtWidgets.QTabWidget(parent=self.centralwidget)
        self.tabWidget.setGeometry(QtCore.QRect(0, 0, 911, 571))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.tabWidget.setFont(font)
        self.tabWidget.setObjectName("tabWidget")
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.add_expense_btn = QtWidgets.QPushButton(parent=self.tab)
        self.add_expense_btn.setGeometry(QtCore.QRect(10, 470, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.add_expense_btn.setFont(font)
        self.add_expense_btn.setObjectName("add_expense_btn")
        self.add_gain_btn = QtWidgets.QPushButton(parent=self.tab)
        self.add_gain_btn.setGeometry(QtCore.QRect(460, 470, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.add_gain_btn.setFont(font)
        self.add_gain_btn.setObjectName("add_gain_btn")
        self.total = QtWidgets.QLineEdit(parent=self.tab)
        self.total.setEnabled(False)
        self.total.setGeometry(QtCore.QRect(70, 510, 821, 20))
        self.total.setObjectName("total")
        self.label_3 = QtWidgets.QLabel(parent=self.tab)
        self.label_3.setGeometry(QtCore.QRect(190, 210, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.save_btn = QtWidgets.QPushButton(parent=self.tab)
        self.save_btn.setGeometry(QtCore.QRect(660, 120, 141, 41))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.save_btn.setFont(font)
        self.save_btn.setObjectName("save_btn")
        self.label_4 = QtWidgets.QLabel(parent=self.tab)
        self.label_4.setGeometry(QtCore.QRect(660, 210, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.label_4.setFont(font)
        self.label_4.setObjectName("label_4")
        self.gains_table = QtWidgets.QTableWidget(parent=self.tab)
        self.gains_table.setGeometry(QtCore.QRect(460, 250, 430, 211))
        self.gains_table.setObjectName("gains_table")
        self.gains_table.setColumnCount(4)
        self.gains_table.setRowCount(0)
        item = QtWidgets.QTableWidgetItem()
        self.gains_table.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.gains_table.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.gains_table.setHorizontalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.gains_table.setHorizontalHeaderItem(3, item)
        self.del_gain_btn = QtWidgets.QPushButton(parent=self.tab)
        self.del_gain_btn.setGeometry(QtCore.QRect(740, 470, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.del_gain_btn.setFont(font)
        self.del_gain_btn.setObjectName("del_gain_btn")
        self.expenses_table = QtWidgets.QTableWidget(parent=self.tab)
        self.expenses_table.setGeometry(QtCore.QRect(10, 250, 430, 211))
        self.expenses_table.setObjectName("expenses_table")
        self.expenses_table.setColumnCount(4)
        self.expenses_table.setRowCount(0)
        item = QtWidgets.QTableWidgetItem()
        self.expenses_table.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.expenses_table.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.expenses_table.setHorizontalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.expenses_table.setHorizontalHeaderItem(3, item)
        self.load_btn = QtWidgets.QPushButton(parent=self.tab)
        self.load_btn.setGeometry(QtCore.QRect(660, 50, 141, 41))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.load_btn.setFont(font)
        self.load_btn.setObjectName("load_btn")
        self.del_expense_btn = QtWidgets.QPushButton(parent=self.tab)
        self.del_expense_btn.setGeometry(QtCore.QRect(290, 470, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.del_expense_btn.setFont(font)
        self.del_expense_btn.setObjectName("del_expense_btn")
        self.calendar = QtWidgets.QCalendarWidget(parent=self.tab)
        self.calendar.setGeometry(QtCore.QRect(10, 10, 601, 171))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.calendar.setFont(font)
        self.calendar.setObjectName("calendar")
        self.line = QtWidgets.QFrame(parent=self.tab)
        self.line.setGeometry(QtCore.QRect(440, 210, 20, 281))
        self.line.setFrameShape(QtWidgets.QFrame.Shape.VLine)
        self.line.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line.setObjectName("line")
        self.label_2 = QtWidgets.QLabel(parent=self.tab)
        self.label_2.setGeometry(QtCore.QRect(10, 510, 61, 20))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.tabWidget.addTab(self.tab, "")
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.del_spending_btn = QtWidgets.QPushButton(parent=self.tab_2)
        self.del_spending_btn.setGeometry(QtCore.QRect(290, 270, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.del_spending_btn.setFont(font)
        self.del_spending_btn.setObjectName("del_spending_btn")
        self.add_spending_btn = QtWidgets.QPushButton(parent=self.tab_2)
        self.add_spending_btn.setGeometry(QtCore.QRect(10, 270, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.add_spending_btn.setFont(font)
        self.add_spending_btn.setObjectName("add_spending_btn")
        self.reg_total = QtWidgets.QLineEdit(parent=self.tab_2)
        self.reg_total.setEnabled(False)
        self.reg_total.setGeometry(QtCore.QRect(70, 310, 821, 20))
        self.reg_total.setObjectName("reg_total")
        self.label_8 = QtWidgets.QLabel(parent=self.tab_2)
        self.label_8.setGeometry(QtCore.QRect(660, 10, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.label_8.setFont(font)
        self.label_8.setObjectName("label_8")
        self.del_income_btn = QtWidgets.QPushButton(parent=self.tab_2)
        self.del_income_btn.setGeometry(QtCore.QRect(740, 270, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.del_income_btn.setFont(font)
        self.del_income_btn.setObjectName("del_income_btn")
        self.label_9 = QtWidgets.QLabel(parent=self.tab_2)
        self.label_9.setGeometry(QtCore.QRect(10, 310, 61, 20))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.label_9.setFont(font)
        self.label_9.setObjectName("label_9")
        self.label_10 = QtWidgets.QLabel(parent=self.tab_2)
        self.label_10.setGeometry(QtCore.QRect(190, 10, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.label_10.setFont(font)
        self.label_10.setObjectName("label_10")
        self.income_table = QtWidgets.QTableWidget(parent=self.tab_2)
        self.income_table.setGeometry(QtCore.QRect(460, 50, 430, 211))
        self.income_table.setObjectName("income_table")
        self.income_table.setColumnCount(4)
        self.income_table.setRowCount(0)
        item = QtWidgets.QTableWidgetItem()
        self.income_table.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.income_table.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.income_table.setHorizontalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.income_table.setHorizontalHeaderItem(3, item)
        self.line_3 = QtWidgets.QFrame(parent=self.tab_2)
        self.line_3.setGeometry(QtCore.QRect(440, 10, 20, 281))
        self.line_3.setFrameShape(QtWidgets.QFrame.Shape.VLine)
        self.line_3.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_3.setObjectName("line_3")
        self.add_income_btn = QtWidgets.QPushButton(parent=self.tab_2)
        self.add_income_btn.setGeometry(QtCore.QRect(460, 270, 150, 23))
        font = QtGui.QFont()
        font.setPointSize(9)
        self.add_income_btn.setFont(font)
        self.add_income_btn.setObjectName("add_income_btn")
        self.spending_table = QtWidgets.QTableWidget(parent=self.tab_2)
        self.spending_table.setGeometry(QtCore.QRect(10, 50, 430, 211))
        self.spending_table.setObjectName("spending_table")
        self.spending_table.setColumnCount(4)
        self.spending_table.setRowCount(0)
        item = QtWidgets.QTableWidgetItem()
        self.spending_table.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.spending_table.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.spending_table.setHorizontalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.spending_table.setHorizontalHeaderItem(3, item)
        self.tabWidget.addTab(self.tab_2, "")
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(parent=MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 905, 21))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(parent=MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)

        self.retranslateUi(MainWindow)
        self.tabWidget.setCurrentIndex(0)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Калькулятор Финансов"))
        self.add_expense_btn.setText(_translate("MainWindow", "Добавить"))
        self.add_gain_btn.setText(_translate("MainWindow", "Добавить"))
        self.label_3.setText(_translate("MainWindow", "Расходы"))
        self.save_btn.setText(_translate("MainWindow", "Сохранить"))
        self.label_4.setText(_translate("MainWindow", "Доходы"))
        item = self.gains_table.horizontalHeaderItem(0)
        item.setText(_translate("MainWindow", "Имя"))
        item = self.gains_table.horizontalHeaderItem(1)
        item.setText(_translate("MainWindow", "Категория"))
        item = self.gains_table.horizontalHeaderItem(2)
        item.setText(_translate("MainWindow", "Сумма"))
        item = self.gains_table.horizontalHeaderItem(3)
        item.setText(_translate("MainWindow", "Примечание"))
        self.del_gain_btn.setText(_translate("MainWindow", "Удалить"))
        item = self.expenses_table.horizontalHeaderItem(0)
        item.setText(_translate("MainWindow", "Имя"))
        item = self.expenses_table.horizontalHeaderItem(1)
        item.setText(_translate("MainWindow", "Категория"))
        item = self.expenses_table.horizontalHeaderItem(2)
        item.setText(_translate("MainWindow", "Сумма"))
        item = self.expenses_table.horizontalHeaderItem(3)
        item.setText(_translate("MainWindow", "Примечание"))
        self.load_btn.setText(_translate("MainWindow", "Загрузить"))
        self.del_expense_btn.setText(_translate("MainWindow", "Удалить"))
        self.label_2.setText(_translate("MainWindow", "Итого:"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab), _translate("MainWindow", "Журнал"))
        self.del_spending_btn.setText(_translate("MainWindow", "Удалить"))
        self.add_spending_btn.setText(_translate("MainWindow", "Добавить"))
        self.label_8.setText(_translate("MainWindow", "Доходы"))
        self.del_income_btn.setText(_translate("MainWindow", "Удалить"))
        self.label_9.setText(_translate("MainWindow", "Итого :"))
        self.label_10.setText(_translate("MainWindow", "Расходы"))
        item = self.income_table.horizontalHeaderItem(0)
        item.setText(_translate("MainWindow", "Имя"))
        item = self.income_table.horizontalHeaderItem(1)
        item.setText(_translate("MainWindow", "Категория"))
        item = self.income_table.horizontalHeaderItem(2)
        item.setText(_translate("MainWindow", "Сумма"))
        item = self.income_table.horizontalHeaderItem(3)
        item.setText(_translate("MainWindow", "Период"))
        self.add_income_btn.setText(_translate("MainWindow", "Добавить"))
        item = self.spending_table.horizontalHeaderItem(0)
        item.setText(_translate("MainWindow", "Имя"))
        item = self.spending_table.horizontalHeaderItem(1)
        item.setText(_translate("MainWindow", "Категория"))
        item = self.spending_table.horizontalHeaderItem(2)
        item.setText(_translate("MainWindow", "Сумма"))
        item = self.spending_table.horizontalHeaderItem(3)
        item.setText(_translate("MainWindow", "Период"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_2), _translate("MainWindow", "Регулярные платежи"))
